From stable-bounces@linux.kernel.org  Sat Oct  8 21:14:33 2005
Date: Sat, 08 Oct 2005 21:14:34 -0700 (PDT)
To: stable@kernel.org
From: "David S. Miller" <davem@davemloft.net>
Cc: 
Subject: [SPARC64]: Fix compile error in irq.c

From: Sven Hartge <hartge@ds9.argh.org>

irq.c is missing the inclusion of asm/io.h, which causes
readb() and writeb() the be undefined.

Signed-off-by: Sven Hartge <hartge@ds9.argh.org>
Signed-off-by: David S. Miller <davem@davemloft.net>
Signed-off-by: Chris Wright <chrisw@osdl.org>
---
 arch/sparc64/kernel/irq.c |    1 +
 1 files changed, 1 insertion(+)

Index: linux-2.6.13.y/arch/sparc64/kernel/irq.c
===================================================================
--- linux-2.6.13.y.orig/arch/sparc64/kernel/irq.c
+++ linux-2.6.13.y/arch/sparc64/kernel/irq.c
@@ -27,6 +27,7 @@
 #include <asm/atomic.h>
 #include <asm/system.h>
 #include <asm/irq.h>
+#include <asm/io.h>
 #include <asm/sbus.h>
 #include <asm/iommu.h>
 #include <asm/upa.h>
